### PR TITLE
Clarify DB connection method

### PR DESCRIPTION
**Pull request by lorentzlasson**
Prep for #530 

Making it more clear that the host parameter is unused when running on GCP. Removed the env from GCP along for :sweden: DEV and will do the same for the other environments if it works

We will likely want to adopt the full DB URL that prisma is using in a next step

---

Tested on devices

- [ ] Desktop 💻
- [ ] Mobile 📱

Tests

- [ ] All tests are running ✔️
- [ ] Test are updated 🧪
- [ ] Code Review 👩‍💻
- [ ] QA 👌

⏲️ Time spent on CR:

⏲️ Time spent on QA:

### CODE REVIEW

File: src/custom_modules/DAO.ts [hunk 1]
```diff
@@ -60,19 +59,23 @@ export const DAO = {
...
       timezone: "+00:00",
     };
-    if (process.env.K_SERVICE != null)
-      args["socketPath"] = `${dbSocketPath}/${process.env.CLOUD_SQL_CONNECTION_NAME}`;
-    else args["host"] = "127.0.0.1";
+
+    // Using Unix sockets to connect to cloud SQL is the default way
+    // https://cloud.google.com/sql/docs/mysql/connect-run#connect-unix-socket
+    if (process.env.K_SERVICE != null) {
+      args["socketPath"] = `cloudsql/${process.env.CLOUD_SQL_CONNECTION_NAME}`;
```
Comment (adamalfredsson) [inline src/custom_modules/DAO.ts:74]: Does it matter if the socketPath does not have a leading `/` after this refactoring?

File: src/custom_modules/DAO.ts [hunk 2]
```diff
@@ -60,19 +59,23 @@ export const DAO = {
    * @param {function} cb Callback for when DAO has been sucessfully set up
    */
   connect: async function (cb) {
-    const dbSocketPath = process.env.DB_SOCKET_PATH || "/cloudsql";
```
Comment (lorentzlasson) [inline src/custom_modules/DAO.ts:63]: Don't really see a reason to support a dynamic path to the unix socket

**Review by adamalfredsson — APPROVED** [review]
Looks cleaner! I'm not sure, what is `K_SERVICE` even? Didn't find any documentation about it, could be perhaps a nice addition!

**Comment by lorentzlasson** [comment]
> Looks cleaner! I'm not sure, what is `K_SERVICE` even? Didn't find any documentation about it, could be perhaps a nice addition!

>  The name of the Cloud Run service being run.

https://cloud.google.com/run/docs/container-contract#services-env-vars

I can add a link in a code comment as well :+1:

File: src/custom_modules/DAO.ts (hunk 1 above)
Comment (lorentzlasson) [reply src/custom_modules/DAO.ts]: Yes :grimacing: Added it

**Pull request merged**